.github/workflows/release-master.yml

```diff
@@ -5,55 +5,61 @@ on:
   push:
     branches:
       - master
+    tags:
+      - v*
 
 jobs:
   golint:
-    name: vet fmt and lint
-    runs-on: ubuntu-20.04
+    name: Lint
+    runs-on: ubuntu-latest
     container: golang:1.13.7
     steps:
-    - uses: actions/checkout@v2.3.3
-    - name: Install golint
-      run: go get -u golang.org/x/lint/golint
-    - run: make lint
-      shell: bash
+      - uses: actions/checkout@v2.3.3
+      - name: Install golint
+        run: go get -u golang.org/x/lint/golint
+      - run: make lint
+        shell: bash
+
   gotest:
-    name: go test
-    runs-on: ubuntu-20.04
+    name: Test
+    runs-on: ubuntu-latest
     container: golang:1.13.7
     steps:
-    - uses: actions/checkout@v2.3.3
-    - run: make test
+      - uses: actions/checkout@v2.3.3
+      - run: make test
+
+  # Job that compiles multi-arch binaries and pushes multi-arch docker images.
   docker_build:
-    name: docker_build
-    runs-on: ubuntu-20.04
-    container:
-      image: docker:20.10.5
-      options: -t
-    # Setting up dind service container
-    services:
-      docker:
-        image: docker:20.10.5-dind
-        env:
-          DOCKER_DRIVER: overlay
-          DOCKER_HOST: tcp://localhost:2375
```

Comment:
Cool! I didn't realise this was not needed with the docker actions.

Reply:
Yeah, if you aren't using a container for the entire job, the ubuntu VM comes with docker already available.